torch/csrc/jit/codegen/cuda/ir_interface_nodes.h

```diff
@@ -261,15 +261,24 @@ struct TORCH_CUDA_API TensorView : public Val {
   // Reorder axes according to old2new[old_pos] = new_pos
   TensorView* reorder(const std::unordered_map<int, int>& old2new);
 
-  /*
-   * WARNING: Does not return this TensorView, returns a new tensorview consumed
-   * to create this!! Take reduction axes out of this domain, and create a new
-   * domain. New domain will be used to create this domain. For example: TV1[I0,
-   * I1] = TV0[I0, R0, R1, I1] TV0->rfactor({1}) TV0 is transformed to ->
-   * TV0[I0, R1, I1] The TensorView returned is: TV2[I0, R0, I3, I1] The
-   * reduction will now beset as: TV1[I0, R1, I1] = TV2[I0, R0, I3, I1] TV0[I0,
-   * I1] = TV1[I0, R1, I1]
-   */
+  // WARNING: Does not return this TensorView, returns a new tensorview consumed
+  // to create this!!
+  //
+  // Take reduction axes out of this domain, and create a new
+  // domain. New domain will be used to create this domain.
+  //
+  // For example:
+  //  TV1[I0, I1] = TV0[I0, R0, R1, I1]
+  //
+  // After:
+  //  TV0->rfactor({1}), TV0 is transformed to -> TV0[I0, R1, I1]
+  //
+  // The TensorView returned is: TV2[I0, R0, I3, I1]
```

Comment:
Made a few corrections to this comment, there were some mistakes you pointed out later in the PR.
```
  // For example:
  //  TV1[I0, R1, R2, I3] = TV0[I0, I1, I2, I3]
  //
  // After:
  //  TV1->rfactor({1}), TV1 is transformed to -> TV1[I0, R2, I3]
  //
  // The TensorView returned is: TV2[I0, R1, I2, I3]
  //
  // The reduction will now beset as:
  //  TV2[I0, R1, I2, I3] = TV0[I0, I1, I2, I3]
  //  TV2[I0, R2, I3] = TV2[I0, R1, I2, I3]
  //
```

Clang format I guess changes formatting of block comments, but not line comments? Is that why clang format won't reformat this comment?

Reply:
Shouldn't the last expression be `TV1[I0, R2, I3] = TV2[I0, R1, I2, I3]` ?

Reply:
Yeah, you're right.